### PR TITLE
Add host subnetMask; managementServerIp.

### LINKED ISSUE #120: Expand network attributes of the Host model
When a vSphere provider is added, the UI leverages the inventory to list the hosts and allow selecting a migration network.
By default, the migration traffic will go over the management network, but the UI cannot display that information. However, the vSphere API exposes the IP address of the ESXi host used for management:

`MOR:HostSystem.summary.managementServerIp`.

If the inventory contained that attribute, the UI could then identify the default network used for migration, by doing some IP calculation based on the IP addresses and netmasks of the pNICs. The netmask is available as:

`MOR:HostSystem.configManager.networkSystem.networkInfo.vnic[*].spec.ip.subnetMask`

### PR DESCRIPTION
**Pull request by jortel**
Add host subnetMask; managementServerIp.

closes #120

### CODE REVIEW

File: pkg/controller/provider/model/vsphere/model.go [hunk 1]
```diff
@@ -231,21 +231,22 @@ type Cluster struct {
...
+	Networks           []Ref       `sql:""`
+	Datastores         []Ref       `sql:""`
+	Vms                []Ref       `sql:""`
 }
 
 type HostNetwork struct {
-	PNICs      []PNIC      `json:"vNICs"`
-	VNICs      []VNIC      `json:"pNICs"`
+	PNICs      []PNIC      `json:"pNICs"`
+	VNICs      []VNIC      `json:"vNICs"`
```
Comment (jortel) [inline pkg/controller/provider/model/vsphere/model.go:249]: Unrelated: Noticed tags were backwards here and fixed.

**Review by mansam — APPROVED** [review]

**Comment by mturley** [comment]
@fdupont-redhat @vconzola did we want to also display the subnet mask in the UI on each VMware host network? I think it may have been in an earlier mockup.

**Comment by fdupont-redhat** [comment]
@mturley Without the network mask, showing the network IP address doesn't make sense.
Say that the host IP address is `10.1.0.24`. Without the netmask you don't know if belongs to `10.1.0.0/16` or `10.0.0.0/8`.
From a UI point of view, the slash notation is shorter: `255.255.0.0` becomes `/16`. And it's a common notation.
Most IP address manipulation libraries can switch from one notation to the other.

**Comment by mturley** [comment]
@fdupont-redhat ok thanks, I'll open an issue to follow up on that.

**Pull request merged**